### PR TITLE
[IMP][14.0] partner_email_check: updated module name

### PR DESCRIPTION
**Pull request by AshishHirapara**
Updated name and description of the module
name update: from - Email Format Checker
to
Partner Email Format and Duplicate Checker

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
Improves searchability of module function

@rousseldenis merge?

**Comment by francesco-ooops** [comment]
@rafaelbn can you take a look? thanks!

**Comment by rafaelbn** [comment]
I don't use this module maybe @janverb or @Abranes could review it 😄

**Comment by francesco-ooops** [comment]
@rafaelbn it's no big change, it's just a clarification of what this module does, to make it easier to find :)

**Comment by Abranes** [comment]
@rafaelbn @francesco-ooops I think that this modification of the name shouldn't be made because making the email unique is something optional in the module and I don't see it correct to indicate it in the name itself, it's enough to indicate it in the readme.

**Comment by francesco-ooops** [comment]
@Abranes yes, but at the same time also checking the validity of the email is optional (all features of this module are optional and managed in config). 

I asked for the name to be expanded just because I was looking for a module to avoid duplicate email and didn't find with keyword "duplicate" which is a quite useful feature of this module. 

So I don't think having more information in the module title hurts anyone, it's good for searching, then there is always readme :)

**Comment by Abranes** [comment]
@francesco-ooops maybe we can add this information in the module summary (the summary is shown in the repository's available addons table)

File: partner_email_check/__manifest__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 {
-    "name": "Email Format Checker",
+    "name": "Partner Email Format and Duplicate Checker",
```
Comment (francesco-ooops) [inline partner_email_check/__manifest__.py:5]: @AshishHirapara remove

File: partner_email_check/__manifest__.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
     "version": "14.0.1.1.1",
     "summary": "Validate email address field",
```
Comment (francesco-ooops) [inline partner_email_check/__manifest__.py:7]: "summary": "Normalize, validate and/or prevent duplicates in email address field,",

**Review by francesco-ooops — CHANGES_REQUESTED** [review]

**Pull request closed without merging**